### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: object/src/file.rs [hunk 1]
```diff
@@ -103,7 +103,7 @@ impl<D, T> OpenFileOptions<D, T> {
     }
 
     /// Set the transfer syntax index to use when reading the file.
-    pub fn tranfer_syntax_index<Tr>(self, ts_index: Tr) -> OpenFileOptions<D, Tr>
+    pub fn transfer_syntax_index<Tr>(self, ts_index: Tr) -> OpenFileOptions<D, Tr>
```
Comment (Enet4) [inline object/src/file.rs:106]: Changing the name of this public function would be a breaking change now. We can 1) postpone this change until the time we aim for 0.9, or 2) re-add `tranfer_syntax_index` as a deprecated function which calls `transfer_syntax_index` underneath.

Comment (jayvdb) [reply object/src/file.rs:106]: Will re-add as deprecated.  I should have thought of that!

File: object/src/file.rs [hunk 2]
```diff
@@ -116,6 +116,15 @@ impl<D, T> OpenFileOptions<D, T> {
         }
     }
 
+    /// Set the transfer syntax index to use when reading the file.
+    #[deprecated(since="0.8.0", note="please use `transfer_syntax_index` instead")]
```
Comment (jayvdb) [inline object/src/file.rs:120]: should this be 0.9.0 ?

Comment (Enet4) [reply object/src/file.rs:120]: We're very likely doing v0.8.1 next.

**Review by Enet4 — APPROVED** [review]
:+1: Thanks.

**Pull request merged**